src/app/[category]/[product]/page.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import styles from './page.module.sass'
+
+import imageProduct from '@/assets/marktwoheadphones/image-product.jpg'
+import imageGallery1 from '@/assets/marktwoheadphones/image-gallery-1.jpg'
+import imageGallery2 from '@/assets/marktwoheadphones/image-gallery-2.jpg'
+import imageGallery3 from '@/assets/marktwoheadphones/image-gallery-3.jpg'
+
+import imageRecommendation1 from '@/assets/recommendations/image-xx99-mark-one-headphones.jpg'
+import imageRecommendation2 from '@/assets/recommendations/image-xx59-headphones.jpg'
+import imageRecommendation3 from '@/assets/recommendations/image-zx9-speaker.jpg'
+import BackButton from '@/components/PDP/BackButton'
+import ImageAndDescription from '@/components/PDP/ImageAndDescription'
+import FeaturesAndBox from '@/components/PDP/FeaturesAndBox'
+import ImageGallery from '@/components/PDP/ImageGallery'
+import Recommendations from '@/components/PDP/Recommendations'
+
+const fetchedData = {
```

Comment:
could please move this to a new file? eg: mockData

Reply:
Moved mock fetchedData object to separate file mockRequests.ts